cdc/sink/ddlsink/factory/factory.go

```diff
@@ -59,6 +61,9 @@ func New(
 		return mysql.NewDDLSink(ctx, changefeedID, sinkURI, cfg)
 	case sink.S3Scheme, sink.FileScheme, sink.GCSScheme, sink.GSScheme, sink.AzblobScheme, sink.AzureScheme, sink.CloudStorageNoopScheme:
 		return cloudstorage.NewDDLSink(ctx, changefeedID, sinkURI, cfg)
+	case sink.PulsarScheme:
```

Comment:
is there `PulsarSSLScheme` ?

Reply:
You are right，it should add ‘PulsarSSLScheme’